contributor-license-agreements/LogSearch_Individual_Contributor_License_Agreement.md

```diff
@@ -1,210 +1,149 @@
-shall not apply and are hereby excluded.
-
-6.2 This Agreement sets out the entire agreement between You and Us for Your
-Contributions to Us and overrides all other agreements or understandings.
-Nothing contained in this Agreement shall be construed to make either one of 
-us an agent, partner, joint venturer, or representative of the other for any purpose. 
-
-6.3 If You or We assign the rights or obligations received through this
-Agreement to a third party, as a condition of the assignment, that third party
-must agree in writing to abide by all the rights and obligations in the
-Agreement.
-
-6.4 The failure of either party to require performance by the other party of
-any provision of this Agreement in one situation shall not affect the right of
-a party to require such performance at any time in the future. A waiver of
-performance under a provision in one situation shall not be considered a
-waiver of the performance of the provision in the future or a waiver of the
-provision in its entirety.
-
-6.5 If any provision of this Agreement is found void and unenforceable, such
-provision will be replaced to the extent possible with a provision that comes
-closest to the meaning of the original provision and which is enforceable. The
-terms and conditions set forth in this Agreement shall apply notwithstanding
-any failure of essential purpose of this Agreement or any limited remedy to
-the maximum extent possible under law.
-
-6.6  You and We intend that no term of the contract made by this Agreement 
-may be enforced by any person who is not a party to it and reserve the 
-right to agree to rescind or vary this Agreement without the consent of any 
-other person. The provisions of the Contracts (Rights of Third Parties) Act 1999 
-shall be and hereby are excluded.
-
-You
-
-________________________
-
-Name: __________________
-
-Address: ________________
-
-________________________
-
-Us
-
-The LogSearch Project (https://github.com/logsearch) by City Index Ltd.
-
-Name: Mike Lear
-
-Title: Chief Information Officer
-
-Address: Park House, 16 Finsbury Circus, London, EC2M 7EB, UNITED KINGDOM
-
-Based on Harmony (HA-CLA-I-OSI) Version 1.0
+```
+Thank you for your interest in The Logsearch Project (the "Project"); 
+run by GAIN Capital UK Ltd; whoes source code is published at http://github.com/logsearch.
+
+In order to clarify the intellectual property license
+granted with Contributions from any person or entity, the Project
+must have a Contributor License Agreement ("CLA") on file that has
+been signed by each Contributor, indicating agreement to the license
+terms below. This license is for your protection as a Contributor as
+well as the protection of the Project and its users; it does not
+change your rights to use your own Contributions for any other purpose.
+If you have not already done so, please complete and sign, then scan
+and email a pdf file of this Agreement to david.laing@labs.cityindex.com.
+
```

Comment:
@mrdavidlaing this email address looks wrong. Should we create a new email address: opensource@gaincapital.com ?